packages/platform-sdk-profiles/src/drivers/memory/services/fee-service.ts

```diff
@@ -31,6 +31,8 @@ export class FeeService implements IFeeService {
 	public async sync(profile: IProfile, coin: string, network: string): Promise<void> {
 		const instance: Coins.Coin = profile.coins().push(coin, network);
 
+		// @TODO: remove this in a refactor
```

Comment:
This will be addressed in https://github.com/ArkEcosystem/platform-sdk/pull/1399